### ISSUE TITLE
When exporting, coordinates are not saved in json if they have the 0 value.

### ISSUE THREAD

**Issue by ambs** [UnderwaterApps/overlap2d#374]
Basically, place an object at 0,0
export it.
the json will not include x, y position.

Cheers
ambs

PS: I am working in a minimal code to import overlap2d scene into MonoGame. Will be happy to share the result in case you wish to incorporate it somehow.

**Comment by azakhary** [comment]
The JSON tries to be smaller (optimize) by not putting in default values (for x,y it's 0,0) 
Is it much trouble? we can change it to not optimize on this, if it is a trouble.

**Comment by ambs** [comment]
Right, that is true, but requires that game frameworks parsing the JSON file look for the existence of that key, and assign the default value if not present.

I can't say it is a trouble, as I already worked it around in my code, but I think it would be easier for newcomers if some keys are always available.

anyway, be free to go in the direction you prefer.
I just reported it because I wasn't sure if this behavior was known and desired.

Best ;)
Alberto
